GOVERNANCE.md

```diff
@@ -165,6 +174,81 @@ We have a reasonably liberal approach to code review and merging. We value quick
 - All code must go through Pull Requests (PR) and pass status checks before being merged. If a PR is merged that breaks `main` due to the branch not being up-to-date, then it should either be reverted or a quick fix merged as a separate PR.
 - If a PR is against code you have previously committed and is either small changes, bug fixes, or refactors, then you're free to merge it without any review. However, if you need more confidence in your changes, you can wait for approval from another maintainer or core contributor.
 
+## Financial Contributions
+
+Biome is in the fortunate position that people care enough about it that they want to invest in its future financially. However, with money comes responsibility, so we are setting the following around
+
+There are two ways of funding Biome and its development, though we greatly prefer the first.
+
+### Sponsorship
+
+Sponsorship is the first and foremost way of making financial contributions to Biome. Sponsorship can be
+done through a one-time donation, or through recurring donations. We offer incentives for certain levels of recurring donations.
+
+To learn more about sponsorship incentives, or to make a donation, please visit our Open Collective page: https://opencollective.com/biome
+
+### Community-Funded Bounties
+
+Bounties are the second approach towards funding Biome development. Unlike a sponsorship, bounties are directed towards the completion of a specific task. Tasks include, but are not limited to, the development of a feature or the fixing of a bug.
+
+Compared to sponsorship, bounties come with a few risks, both for Biome as a project, as well as the persons or entities offering and collecting the bounty. To minimize these risks, we only sanction bounties under the following conditions:
+
+- Bounties must be pledged through Polar.sh on an issue that is opened in our issue tracker.
+  - Unless otherwise specified, Biome reserves 30% of the payment amount, meaning contributors receive 70% of the pledged amount.
```

Comment:
Do you know if 70/30 is the usual amount for bounties?

Reply:
To be honest, I don't. I would be leaning to either 70/30 or 80/20, but it's hard to say which should be preferred. And the range as a whole is more a gut-feeling than anything.